.eslintrc.cjs

```diff
@@ -17,10 +20,16 @@ module.exports = {
     'comma-dangle': ['error', 'always-multiline'],
     'no-trailing-spaces': ['error'],
     'eol-last': ['error', 'always'],
+    'node/no-process-env': ['error'],
```

Comment:
The other interesting part is the `rules` section, which is where we specify our own rules.

Each rule has a name (e.g. `eol-last` or `node/no-process-env` - rules from plugins have the `plugin-name/rule-name`, built in ones just `name`) and then the value is an array containing `[severity, configuration]`.

`severity` is either `error`, `warn`, or `off`. Most of the time you should just make everything `error` or `off` because people tend to ignore warnings and soon you have 500 warnings and they're all meaningless.

Then the configuration is dependent on the rule, if the rule even takes configuration at all. Of the ones I highlighted here, only `eol-last` takes configuration, which you can find documentation for [online](https://eslint.org/docs/rules/eol-last#require-or-disallow-newline-at-the-end-of-files-eol-last).

Reply:
What kind of warnings does the linter pump out? Are they not important, thus why people ignore them?

Reply:
Well you can configure any of the rules to either be "warning" or "error". Typically you'd configure "error" to cause CI/build to fail, but warnings would be allowed through.